src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,17 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => {
+  const { title, completed, user } = todo;
+
+  return (
+    <article className={cn('TodoInfo', {
+      'TodoInfo--completed': completed === true,
+    })}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+
+      <UserInfo user={user} />
```

Comment:
<img width="623" alt="image" src="https://github.com/mate-academy/react_static-list-of-todos-js/assets/86789161/896fda5c-76db-4088-b72b-2cdcff6ebf39">

You should not render this part if user doesn't exist